shared/src/main/scala/org/atnos/eff/IntoPoly.scala

```diff
@@ -15,7 +15,10 @@ import Eff._
  * Note: some implicit definitions are probably missing in some cases
  */
 trait IntoPoly[R, U] {
-  def apply[A](e: Eff[R, A]): Eff[U, A]
+  val unionInto: UnionInto[R, U]
```

Comment:
:+1: I was considering making an issue on this; the fact that `IntoPoly` can't really have inductive instances. Nice to see it fixed.